app/assets/javascripts/primer_view_components.js.map

```diff
@@ -1 +1 @@
AAK4P,MAAM5E,EAAM,IACzBM,EAAOtL,KAAK4P,MAAMxE,EAAQ,IAEhC,OAAIJ,GAAO,IACF,GAAGJ,OAAOU,EAAM,KACdwE,GAAM,GACR,GAAGlF,OAAOI,EAAK,KACb6E,GAAO,GACT,GAAGjF,OAAOkF,EAAI,KACZD,EAAM,EACR,GAAGjF,OAAOiF,EAAK,KAEf,KAIXnQ,aACE,IAAIyM,EAASQ,IAAe,QAAU,QAta1C,IAAoBzB,EA4ahB,OA5agBA,EAwaArL,KAAKqL,MAvaX,IAAI4B,MACLqD,mBAAqBjF,EAAKiF,mBAuajChE,GA7bN,WACE,GAAsB,OAAlBY,EACF,OAAOA,EAGT,MAAMH,EAAYI,IAElB,GAAIJ,EAAW,CACb,MAAMC,EAASD,EAAUT,OAAO,IAAIW,KAAK,IAEzC,OADAC,IAAkBF,EAAOrN,MAAM,OACxBuN,EAEP,OAAO,EAibKqD,GAAoB,OAAS,OAGlCvF,EAAShL,KAAKqL,KAAMiB,GAG7BzM,aACE,MAAMkN,EAAYyD,IAElB,OAAIzD,EACKA,EAAUT,OAAOtM,KAAKqL,MAEtBL,EAAShL,KAAKqL,KAAM,YAMjC,SAAS+E,EAAmBb,EAAQ/H,EAAOiJ,GACzC,MAAM1D,EAxbR,SAA4BwC,EAAQlD,GAClC,GAAI,SAAUtF,QAAU,uBAAwBA,OAAOwF,KACrD,IAGE,OAAO,IAAIA,KAAKmE,mBAAmBnB,EAAQlD,GAC3C,MAAOK,GACP,KAAMA,aAAaC,YACjB,MAAMD,GAgbMiE,CAAmBpB,EAAQ,CAC3CV,QAAS,SAGX,OAAI9B,EACKA,EAAUT,OAAO9E,EAAOiJ,GAWnC,SAA8BjJ,EAAOiJ,GACnC,GAAc,IAAVjJ,EACF,OAAQiJ,GACN,IAAK,OACL,IAAK,UACL,IAAK,QACL,IAAK,OACH,MAAO,QAAQ1F,OAAO0F,GAExB,IAAK,MACH,MAAO,QAET,IAAK,OACL,IAAK,SACH,MAAO,QAAQ1F,OAAO0F,EAAM,KAE9B,IAAK,SACH,MAAO,WAEN,GAAc,IAAVjJ,EACT,OAAQiJ,GACN,IAAK,OACL,IAAK,UACL,IAAK,QACL,IAAK,OACH,MAAO,QAAQ1F,OAAO0F,GAExB,IAAK,MACH,MAAO,WAET,IAAK,OACL,IAAK,SACL,IAAK,SACH,MAAO,QAAQ1F,OAAO0F,QAErB,IAAe,IAAXjJ,EACT,OAAQiJ,GACN,IAAK,OACL,IAAK,UACL,IAAK,QACL,IAAK,OACH,MAAO,QAAQ1F,OAAO0F,GAExB,IAAK,MACH,MAAO,YAET,IAAK,OACL,IAAK,SACL,IAAK,SACH,MAAO,KAAK1F,OAAO0F,EAAM,aAExB,GAAIjJ,EAAQ,EACjB,OAAQiJ,GACN,IAAK,OACL,IAAK,UACL,IAAK,QACL,IAAK,OACL,IAAK,MACL,IAAK,OACL,IAAK,SACL,IAAK,SACH,MAAO,MAAM1F,OAAOvD,EAAO,KAAKuD,OAAO0F,EAAM,UAE5C,GAAIjJ,GAAS,EAClB,OAAQiJ,GACN,IAAK,OACL,IAAK,UACL,IAAK,QACL,IAAK,OACL,IAAK,MACL,IAAK,OACL,IAAK,SACL,IAAK,SACH,MAAO,GAAG1F,QAAQvD,EAAO,KAAKuD,OAAO0F,EAAM,SAIjD,MAAM,IAAI9D,WAAW,uCAAuC5B,OAAO0F,EAAM,MAtFhEG,CAAqBpJ,EAAOiJ,GAyFvC,MAAMD,EAAgBpE,EAAc,CAClCT,KAAM,UACNE,OAAQ,YAGV,MAAMgF,UAA4BtD,EAChC1N,mBACE,MAAMwL,EAAOrL,KAAKqL,KAElB,GAAIA,EACF,OAAO,IAAIiE,EAAajE,EAAM+B,EAAkBpN,OAAOK,WAI3DR,oBACEiR,EAAYC,KAAK/Q,MAEZgR,IACHC,IACAD,EAAsBE,YAAYD,EAAmB,MAGvDvM,MAAMyM,oBAGRtR,uBACE,MAAMuR,EAAKN,EAAYhO,QAAQ9C,OAEnB,IAARoR,GACFN,EAAYO,OAAOD,EAAI,GAGpBN,EAAY/N,QACXiO,IACFM,cAAcN,GACdA,EAAsB,OAQ9B,MAAMF,EAAc,GAEpB,IAAIE,EAGJ,SAASC,IACP,IAAIhG,EAAMsG,EAAGC,EAEb,IAAKD,EAAI,EAAGC,EAAMV,EAAY/N,OAAQwO,EAAIC,EAAKD,IAC7CtG,EAAO6F,EAAYS,GACnBtG,EAAKnD,YAAcmD,EAAK0C,oBAAsB,GAS7C5G,OAAO0C,eAAepE,IAAI,mBAC7B0B,OAAO8J,oBAAsBA,EAC7B9J,OAAO0C,eAAeC,OAAO,gBAAiBmH,IAGhD,MAAMY,UAAuBZ,EAC3BhR,mBACE,MAAMyM,EAAStM,KAAKiB,aAAa,UAC3BoK,EAAOrL,KAAKqL,KAClB,GAAKA,EAEL,MAAe,UAAXiB,EACK,IAAIgD,EAAajE,EAAM+B,EAAkBpN,OAAO0R,eAEhD,IAAIpC,EAAajE,EAAM+B,EAAkBpN,OAAO2R,WAMxD5K,OAAO0C,eAAepE,IAAI,cAC7B0B,OAAO0K,eAAiBA,EACxB1K,OAAO0C,eAAeC,OAAO,WAAY+H,IAG3C,MAAMG,UAAyBf,EAC7BhR,mBACE,MAAMyM,EAAStM,KAAKiB,aAAa,UAC3BoK,EAAOrL,KAAKqL,KAClB,GAAKA,EAEL,MAAe,UAAXiB,EACK,IAAIgD,EAAajE,EAAM+B,EAAkBpN,OAAO6R,iBAEhD,IAAIvC,EAAajE,EAAM+B,EAAkBpN,OAAOmQ,aAMxDpJ,OAAO0C,eAAepE,IAAI,gBAC7B0B,OAAO6K,iBAAmBA,EAC1B7K,OAAO0C,eAAeC,OAAO,aAAckI"}
```

Comment:
Is there a way to get the trailing new line here?

Reply:
not sure, @keithamus do you know?

Reply:
Not easily. Sorry :disappointed: